src/main/java/de/retest/recheck/cli/subcommands/Stage.java

```diff
@@ -0,0 +1,44 @@
+package de.retest.recheck.cli.subcommands;
+
+import java.util.Arrays;
+import java.util.List;
+
+//import de.retest.recheck.ChangeApplier;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+import picocli.CommandLine.Parameters;
+
+@Command( name = "stage", description = "Stage all changes of a certain state and all same changes to other states" )
+public class Stage implements Runnable {
+
+	@Option( names = { "-h", "--help" }, usageHelp = true, hidden = true )
+	private boolean displayHelp;
+
+	@Parameters( description = "Checks to stage the changes off. Leave blank to stage all checks." )
+	private List<String> checks;
+
+	@Override
+	public void run() {
+		try {
+			stageChecks();
+		} catch ( final Exception e ) {
+			System.out.println( "Exception showing diffs: " + e.getMessage() );
+		}
+	}
+
+	public void stageChecks() {
+		System.out.println("TODO Implement!");
+		// TODO Implement staging the given states
+		// and persist that in a separate file
+		// Configuration.ensureLoaded();
+		// File latestReport = ReportFileUtils.getLatestReport();
+		// ChangeApplier applier = new ChangeApplier(checks);
+		// applier.apply(latestReport);
+	}
+
+	public static void main( String... args ) {
```

Comment:
AFAIK only the main entry point needs a `main`, which can be removed from all subcommands (see our CLI impl).